src/main/java/org/onedatashare/server/WebConfiguration.java

```diff
@@ -30,30 +30,31 @@ public void customize(NettyReactiveWebServerFactory factory) {
                 ssl.setEnabled(true);
                 ssl.setKeyStoreType("PKCS12");
                 ssl.setKeyStore("classpath:keystore.p12");
-                ssl.setKeyPassword("asdasd");
+                ssl.setKeyPassword("ODS@DIDCLab");
                 ssl.setKeyAlias("tomcat");
-                ssl.setKeyStorePassword("asdasd");
+                ssl.setKeyStorePassword("ODS@DIDCLab");
                 factory.setSsl(ssl);
             }
         };
     }
     /*
        Below methods are used for redirecting port 8080 to 8443
      */
-    /*@Autowired
+
+    @Autowired
     HttpHandler httpHandler;
 
     WebServer http;
 
     @PostConstruct
     public void start() {
-        ReactiveWebServerFactory factory2 = new NettyReactiveWebServerFactory(8080);
+        ReactiveWebServerFactory factory2 = new NettyReactiveWebServerFactory(80);
         this.http = factory2.getWebServer(this.httpHandler);
         this.http.start();
     }
 
     @PreDestroy
     public void stop() {
         this.http.stop();
-    }*/
+    }
 }
```

Comment:
I dont find any setting for HTTPS (443) in this class. Can you confirm if this is implicitly handled.

If it is, please explain how so that all are aware.

Reply:
# Define a custom port instead of the default 8080
server.port=443
security.require-ssl=true

read application.property